etl/src/main/scala/etl/ETL.scala

```diff
@@ -89,13 +89,16 @@ object ETL extends App {
   }
 
   def processPage(contents: List[Content])(implicit db: DB): Progress = {
-    val progress = contents.foldMap { content =>
+    val existingArticlesIds = db.existingArticlesIds
```

Comment:
This should be `db.existingArticlesIds()` as it has a side effect of calling the DB.